crates/nu-command/tests/commands/table.rs

```diff
@@ -948,7 +952,7 @@ fn test_expand_big_0() {
             "│                  │ │                                 │ │ dependencies │ │             │ ╭──────────┬─────────╮ │ │ │ │",
             "│                  │ │                                 │ │              │ │ openssl     │ │ version  │ 0.10.38 │ │ │ │ │",
             "│                  │ │                                 │ │              │ │             │ │ features │ [list 1 │ │ │ │ │",
-            "│                  │ │                                 │ │              │ │             │ │          │  item]  │ │ │ │ │",
+            "│                  │ │                                 │ │              │ │             │ │          │ item]   │ │ │ │ │",
             "│                  │ │                                 │ │              │ │             │ │ optional │ true    │ │ │ │ │",
```

Comment:
@fdncred notice the difference I was talking about.
I haven't yet fixed it just in case maybe you will think it's all right :smile: 

As I did not yet came to total conclusion myself :)

Reply:
I could live with a space at the end because it's pretty easy to just use `str trim`. That space problem is preferable over the odd table wrapping panic problem. You can always address the space problem later. We have a release coming up on Tuesday. It would be nice to get this in soon if possible, so we have some time to test it.

Reply:
Seems like it's green;
And I've just released `tabled` to `crates.io`.

Reply:
The wasm isn't green. I've asked @cptpiepmatz to help figure out what's going on.